src/background/store.ts

```diff
@@ -17,11 +18,17 @@ function reducer(state: AppState = emptyState, action: Action): AppState {
 
   const responder: Responder<typeof action.type> = responders[action.type]
   const stateUpdates: Partial<AppState> = responder(state, action)
-  return {
+  const nextState = {
     ...state,
     ...stateUpdates,
     mostRecentAction: action
   }
+
+  if (action.type.startsWith('chrome')) {
+    console.log(action.type, (action as any).payload, nextState.focusedWindowId, nextState.tabs)
+  }
```

Comment:
Will this be removed by the production build?

Reply:
Good call, added a guard around that